### PR TITLE
Fips test suite tests

### PR DESCRIPTION
**Pull request by nquarton**

### CODE REVIEW

**Review by jlmahowa-amd — APPROVED** [review]

**Review by mhatrevi — APPROVED** [review]

File: runtime/Cargo.toml [hunk 1]
```diff
@@ -59,3 +59,4 @@ fips_self_test=[]
 no-cfi = ["caliptra-image-verify/no-cfi", "caliptra-drivers/no-cfi"]
 fpga_realtime = ["caliptra-drivers/fpga_realtime"]
 "hw-1.0" = ["caliptra-builder/hw-1.0", "caliptra-drivers/hw-1.0", "caliptra-registers/hw-1.0", "caliptra-kat/hw-1.0","caliptra-cpu/hw-1.0"]
+fips-test-hooks = ["caliptra-drivers/fips-test-hooks"]
```
Comment (jhand2) [inline runtime/Cargo.toml:62]: nit: should this be marked as an optional dependency?

Comment (nquarton) [reply runtime/Cargo.toml:62]: I think I'm missing what you mean here. This adds a feature and propagates it to the drivers crate. But there is not a separate, new dependency here that can be made optional. But please let me know if I'm misunderstanding something.

**Review by rusty1968 — APPROVED** [review]

File: runtime/Cargo.toml (hunk 1 above)
Comment (jhand2) [reply runtime/Cargo.toml:62]: Oh my mistake, I misread this. That's what I get for reviewing on my phone :)

Comment (nquarton) [reply runtime/Cargo.toml:62]: I mean that's what I get for accidentally asking you to do this on vacation. Thanks!

**Pull request merged**